src/conversion/azure/azure_converter.ts

```diff
@@ -0,0 +1,83 @@
+import {NodeSpec, SymbolStore} from '../../graph';
+import {
+  AnyAzureObject,
+  ConverterStore,
+  EntityStore,
+  LocalIpConverter,
+  NetworkInterfaceConverter,
+  PublicIpConverter,
+  SubnetConverter,
+  VirtualNetworkConverter,
+} from '.';
+import {INodeSpecUniverse} from '../contracts';
+
+export class AzureConverter {
+  private readonly converters: ConverterStore;
```

Comment:
Kudos for "private readonly". I am enthusiastic about type safety, intentionality, readability, and defensive driving.

Reply:
Same here. I try to follow the path of immutable types as much as possible. I find it's easier to start that way then intentionally open up instead of having to later lockdown when bugs are located.